queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/Factories/SignatureItemFactoryDEFAULT.cs

```diff
@@ -16,14 +17,15 @@ public class SignatureItemFactoryDEFAULT : SignatureItemFactory
             CurrencyDecimalDigits = 2
         };
         
-        public List<SignaturItem> GetSignaturesForPosReceiptTransaction(string cashBoxId, string receiptId, decimal sumOfPayItems, string previousHash, long ftReceiptCase)
+        public List<SignaturItem> GetSignaturesForPosReceiptTransaction(Guid cashBoxId, Guid receiptId,
+            decimal sumOfPayItems, string receiptHash, long ftReceiptCase)
         {
             var queueInfo = "DEFAULT";
             var receiptCaseInfo = ftReceiptCase.ToString();
 
             var signatures = new List<SignaturItem>
             {
-                CreateQrCodeSignature("www.fiskaltrust.eu", $"{cashBoxId}_{receiptId}_{sumOfPayItems}_{previousHash}")
+                CreateQrCodeSignature("www.fiskaltrust.eu", $"{cashBoxId}_{receiptId}_{receiptHash}_{sumOfPayItems}")
```

Comment:
The `receiptHash` should be the last item not the `sumOfPayItems` 🙏

Also we should call it `previousHash` so that its clear what it is 🤔